packages/product-download-page/partials/release-information/style.module.css

```diff
@@ -1,7 +1,6 @@
 .root {
   background-color: var(--gray-6);
-  padding: 42px 0;
-  margin: 42px 0;
+  padding: 48px 0 128px 0;
```

Comment:
Have used equivalent padding rather than margin in a few places here, to avoid space around the component, which was causing a noticeable gap in some contexts:

![fix-gap-at-page-bottom](https://user-images.githubusercontent.com/4624598/129247795-7af65abc-e722-4330-80e8-a0d02d08c7b7.png)


Have also rounded numbers to multiples of 8 since I know on the design side there has been some form of intent to adopt an 8px grid.